client/state/selectors/is-connected-secondary-network-site.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { isMainSiteOf } from 'state/selectors';
+import { isJetpackSiteSecondaryNetworkSite } from 'state/sites/selectors';
+
+/**
```

Comment:
Hi @tyxla, the use case of this selector is when displaying website where a plugin is installed. If the website is connected to a main site you don't show it in the main list. If it is not connected and has the plugin installed it is shown. You can check the PR that makes use of this selector here: https://github.com/Automattic/wp-calypso/pull/15742/files.